native/android-jsi/proguard-rules.pro

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```

Comment:
same for this file

Reply:
fixed in https://github.com/Nozbe/WatermelonDB/pull/693/commits/2cbf4e017607f44d99e8e8c064ba1f01eab9bce1